### ISSUE TITLE
[3.1] When icon is missing for a collection, just show nothing, rather than img icon and alt text

### ISSUE THREAD

**Issue by themerekat** [Symbiota/Symbiota#1257]
On https://dev002.symbiota.org/collections/search/index.php

![image](https://github.com/BioKIC/Symbiota/assets/16688784/5ab80aac-dbc2-4a54-9790-a45c0625c4ad)
When an icon is missing for a collection, you get an ugly image and text that spills over into the next icons. If an image doesn't exist, it should just be a blank space.

**Comment by themerekat** [comment]
Similarly, when there is no image associated with an occurrence, show nothing, rather than icon and alt text
![image](https://github.com/BioKIC/Symbiota/assets/16688784/adde793d-272d-4398-a026-d88ca614baf1)
E.g., https://dev002.symbiota.org/collections/list.php?db=40&includeothercatnum=1&taxa=Erythranthe+guttata&usethes=1&taxontype=2

**Comment by GregoryPost** [comment]
First issue is addressed in #1263.

@themerekat - Second issue is a symptom of the test environment - the DB has relative links to images that do not exist in the Dev environment - so I don't believe this is an actual bug.
